### PR TITLE
[GraphBolt] Update `to_pyg_data()` to support get batch_size from `seeds`.

### PR DESCRIPTION
**Pull request by yxy235**
## Description

 Update `to_pyg_data()` in `MiniBatch` to support get batch_size from `seeds`.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [ ] I've leverage the [tools](https://docs.google.com/document/d/1iHyj7zlmygKSk5gBPsqIqL5ASPzJSPREaNT_QdsiYA4/edit) to beautify the python and c++ code.
- [ ] The PR is complete and small, read the [Google eng practice (CL equals to PR)](https://google.github.io/eng-practices/review/developer/small-cls.html) to understand more about small PR. In DGL, we consider PRs with less than 200 lines of core code change are small (example, test and documentation could be exempted).
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR
- [ ] If the PR is for a new model/paper, I've updated the example index [here](../examples/README.md).

### CODE REVIEW

**Review by RamonZhou — APPROVED** [review]
LGTM

File: python/dgl/graphbolt/minibatch.py [hunk 1]
```diff
@@ -536,6 +536,11 @@ def to_pyg_data(self):
                 batch_size = len(next(iter(self.node_pairs.values()))[0])
             else:
                 batch_size = len(self.node_pairs[0])
+        elif self.seeds is not None:
+            if isinstance(self.seeds, Dict):
```
Comment (mfbalin) [inline python/dgl/graphbolt/minibatch.py:540]: Do the tests cover the Dict case?

File: tests/python/pytorch/graphbolt/test_minibatch.py [hunk 2]
```diff
@@ -889,6 +889,14 @@ def test_to_pyg_data():
     assert pyg_data.batch_size == expected_batch_size
     assert torch.equal(pyg_data.n_id, expected_n_id)
 
+    # Test with seeds in minibatch.
```
Comment (frozenbugs) [inline tests/python/pytorch/graphbolt/test_minibatch.py:892]: Consider make it a separate test case, such that when clean up in the future it is easier?

**Review by frozenbugs — APPROVED** [review]

**Review by mfbalin — COMMENTED** [review]
LGTM overall.

**Review by mfbalin — APPROVED** [review]

**Review by frozenbugs — APPROVED** [review]

**Pull request merged**